src/commonMain/kotlin/baaahs/shows/HeartbleatShow.kt

```diff
@@ -24,7 +24,8 @@ object HeartbleatShow : Show("Heartbleat") {
             val otherSurfaces = showRunner.allUnusedSurfaces.map { showRunner.getShaderBuffer(it, SolidShader()) }
 
             override fun nextFrame() {
-                var phase = (beatProvider.beat % 1.0) * 3.0f
+                val currentBeat = showRunner.currentBeat - 1
```

Comment:
I'm still iffy on having 1-based `currentBeat`, I suspect we'll be subtracting 1 from it everywhere. But let's go with this and we can change later if we want.

Reply:
Agreed. I'm just gonna make it 0..3 based, this is too confusing lol